src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -155,8 +155,8 @@
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Unix\libc\Interop.close.cs">
-      <Link>Interop\Unix\libc\Interop.close.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.Close.cs">
+      <Link>Interop\Unix\libc\Interop.Close.cs</Link>
```

Comment:
OK, I made another mistake so I fixed this too to at least be consistent with the same file, but I would like to standardize all of the link paths.